web/src/settings_users.js

```diff
@@ -467,6 +469,50 @@ export function set_up_humans() {
     start_data_load();
     section.active.handle_events();
     section.deactivated.handle_events();
+    setting_invites.set_up();
+
+    const userToggler = components.toggle({
+        child_wants_focus: true,
+        values: [
+            {label: $t({defaultMessage: "Users"}), key: "user-list-admin"},
+            {label: $t({defaultMessage: "Deactivated users"}), key: "deactivated-users-admin"},
+            {label: $t({defaultMessage: "Invitations"}), key: "invites-list-admin"},
+        ],
+        callback(_name, key) {
+            if (key !== "user-list-admin") {
+                $("#admin-user-list .user-section-container").hide();
+            } else {
+                $("#admin-user-list .user-section-container").show();
+            }
+
+            if (key === "deactivated-users-admin") {
+                allow_sorting_deactivated_users_list_by_email();
```

Comment:
Why is this function called here? It just returns a value whether to allow sorting by email or not and I don't see any reason to call this function here.

Reply:
Ohh noted , cleaned it .